### DISCUSSION TITLE
Error in command: npx migrate dev

### DISCUSSION THREAD

**Discussion by gholiveira29** [prisma/prisma#17664] [Q&A]
Goodnight!
Could you help me with the following error:

$ npx prisma migrate dev
Environment variables loaded from .env
Prisma schema loaded from prisma\schema.prisma
Datasource "db": SQLite database "dev.db" at "file:./dev.db"

Error: Unable to establish a connection to query-engine-node-api library. It seems that the current architecture x64 is not supported, or that libc is missing from the system.
Details: Unable to require(`C:\Users\guide\OneDrive\Documentos\habities\server\node_modules\@prisma\engines\query_engine-windows.dll.node`)
 Não foi possível encontrar o módulo especificado.
\\?\C:\Users\guide\OneDrive\Documentos\habities\server\node_modules\@prisma\engines\query_engine-windows.dll.node      
[Context: getDmmf]
\\?\C:\Users\guide\OneDrive\Documentos\habities\server\node_modules\@prisma\engines\query_engine-windows.dll.node      
[Context: getDmmf]

Prisma CLI Version : 4.9.0

**Comment by janpio** [comment]
Did you run Prisma successfully before on this machine?

**Reply by nurul3101** [reply]
(Looks related to other ongoing GitHub Discussion: https://github.com/prisma/prisma/discussions/17570)

**Reply by janpio** [reply]
My question is still relevant - although I would also really prefer people only opening one issue so we do not get confused :/

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
